packages/configuration/src/config/vitest.config.ts

```diff
@@ -0,0 +1 @@
+export default 2;
```

Comment:
Why is this needed?

Reply:
Maybe the whole `configuration` package is not useful any more

Reply:
Yes I agree, we should remove it for now. We can always add it back later